File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TaggedAddressBase.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id The ID of the address being tagged.
 */
data class TaggedAddressBase (

        @get:NotNull 
        @JsonProperty("id") val id: Long
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/TaggedAddressBase.scala

package org.openapitools.client.model


case class TaggedAddressBase (
    /* The ID of the address being tagged. */
    _id: Long
)
object TaggedAddressBase {
    def toStringBody(var_id: Object) =
        s"""
        | {
        | "id":$var_id
        | }
        """.stripMargin
}
